Automattic/wp-calypso
# Logged in Profiler: Allow searching pages in dropdown and show empty states

Relates to https://github.com/Automattic/dotcom-forge/issues/9372

## Proposed Changes

* Add disclaimer to the end of the pages list `Searching your top 20 most popular pages.`
* Remove call to backend when user searches the dropdown list
* Forces a `No pages found` item when no pages are found

![image](https://github.com/user-attachments/assets/f71bf4bc-b63d-4c49-a05a-505c30f2fbab)

![image](https://github.com/user-attachments/assets/935e45ea-4fc1-4e66-83b0-7bd30d8bae1d)


## Why are these changes being made?


* To make the experience more predictable and provide clear indications when pages are not available

## Testing Instructions



* Go to `/sites/performance/site`
* Verify `Searching your top 20 most popular pages.` copy is show at the bottom of the list
* Search for a page that doesn't exist and verify `No pages found` is shown

## Pre-merge Checklist
